radicle-httpd/src/api/json.rs

```diff
@@ -91,22 +91,26 @@ pub(crate) fn tree(tree: &Tree, path: &str, stats: &Stats) -> Value {
 
 /// Returns JSON for an `issue`.
 pub(crate) fn issue(id: IssueId, issue: Issue) -> Value {
+    let author: Did = issue.author().into();
+
     json!({
         "id": id.to_string(),
-        "author": issue.author(),
+        "author": author,
         "title": issue.title(),
         "state": issue.state(),
-        "assignees": issue.assigned().collect::<Vec<_>>(),
+        "assignees": issue.assigned().collect::<Dids>(),
```

Comment:
Perhaps `assigned` should return `Did`s instead.

Reply:
I'm trying to change the return type of `assigned` but my rust is too low level it seems, or I have to wrap it inside a Wrapper struct which I think is a PITA.
Can we maybe collect the iterator inside `assigned` into a `Vec<Author>`?

Reply:
I did it here https://github.com/radicle-dev/heartwood/pull/328/files#diff-d8a3305359907b1f57d685f419f888f95c5738c23a3420ca4e7150e9f42cbd4eR156-R158 :)

Reply:
nice 👍

Reply:
Has been reverted, uses what is implemented in #324